revenuecat_examples/purchase_tester/integration_test/app_test.dart

```diff
@@ -8,18 +7,18 @@ import 'package:purchases_flutter/purchases_flutter.dart';
 void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
-  var uuid = UniqueKey().toString();
+  var userId = "integration-test-${DateTime.now().millisecondsSinceEpoch}";
```

Comment:
This was needed after upgrading to xcode 15.2. With the old system, the encoding of some characters didn't match the one returned from the SDK...